pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -668,6 +668,75 @@ func (c *Cluster) LogsFollow(ctx context.Context, name string, out io.Writer) er
 	return c.logs(ctx, name, out, true)
 }
 
+// CollectLogs returns the logs of the specified component.
+func (c *Cluster) CollectLogs(ctx context.Context, name string, dir string) error {
+	conf, err := c.Config(ctx)
+	if err != nil {
+		return err
+	}
+
+	kindPath, err := c.preDownloadKind(ctx)
+	if err != nil {
+		return err
+	}
+	err = exec.WriteToPath(ctx, filepath.Join(dir, conf.Options.Runtime+"-info.txt"), []string{kindPath, "version"})
```

Comment:
```suggestion
	err = exec.WriteToPath(ctx, filepath.Join(dir, conf.Options.Runtime+"-info.txt"), []string{conf.Options.Runtime, "info"})
```

Reply:
kind does not have `kind info`
```
Available Commands:
  build       Build one of [node-image]
  completion  Output shell completion code for the specified shell (bash, zsh or fish)
  create      Creates one of [cluster]
  delete      Deletes one of [cluster]
  export      Exports one of [kubeconfig, logs]
  get         Gets one of [clusters, nodes, kubeconfig]
  help        Help about any command
  load        Loads images into nodes
  version     Prints the kind CLI version
```

Reply:
Oh, sorry, mistaken, it should be `c.runtime`.